### ISSUE TITLE
can we use another Currency Unit

### ISSUE THREAD

**Issue by jordan-huangwei** [hjacobs/kube-resource-report#184]
In China, we need to use CNY to replace USD for AWS billing, so can we use options to choose Currency Unit by his  will

**Comment by hjacobs** [comment]
Yes, supporting a different currency would be nice, but it's not implemented right now.

**Comment by jordan-huangwei** [comment]
do you have any plan to do that?
